src/rard/research/models/antiquarian.py

```diff
@@ -144,9 +143,7 @@ class Meta:
         through="TestimoniumLink",
     )
 
-    bibliography_items = GenericRelation(
-        "BibliographyItem", related_query_name="bibliography_items"
-    )
+    bibliography_items = models.ManyToManyField("BibliographyItem", blank=True)
```

Comment:
```suggestion
    bibliography_items = models.ManyToManyField("BibliographyItem", related_name="antiquarians", blank=True)
```

See comment below re citing authors related name.